bot_python_sdk/store.py

```diff
@@ -1,76 +1,107 @@
 import json
 import os
+from bot_python_sdk.logger import Logger
 
 _actions_file_path = 'storage/actions.json'
 _last_triggered_file_path = 'storage/last_triggered.json'
 _qr_image_path = 'storage/qr.png'
 _configuration_file_path = 'storage/configuration.json'
 _bot_public_key = 'storage/public.pem'
 
+LOCATION = 'Store'
+
 
 class Store:
+    def __init__(self):
+        pass
+
     @staticmethod
     def set_actions(actions):
-        file = open(_actions_file_path, 'w')
-        file.write(json.dumps(actions))
-        file.close()
+        try:
+            with open(_actions_file_path, 'w') as file:
+                file.write(json.dumps(actions))
+        except IOError as io_error:
+            Logger.error(LOCATION, 'Unable to write in {0} path'.format(_actions_file_path))
+            raise io_error.message
```

Comment:
I think you want to raise the error itself here, not just the message

Reply:
i have changed the code in latest build